### ISSUE TITLE
Understand Core blocks and customizations of their editing

### ISSUE THREAD

**Issue by alcurrie** [ampproject/amp-wp#1567]
**As a AMP story author, I want more customization options for my AMP story**
 - [ ] ~_AC 1:_ Explore option to customize font faces (ie. maybe Google fonts)~ _This issue may be blocked by Gutenberg and/or other dependencies so we've separated this AC into a separate issue_ #1643  Related to #1565  - Larger default styles
 - [x] _AC 2:_ For images “caption” blocks are turned off by default with an interface to turn them back on

**Comment by miina** [comment]
Note for dev: Development is blocked by UX/design work.

**Comment by jwold** [comment]
At the moment I'm thinking a simple toggle in the right sidebar for captions, and a dropdown for fonts, should be sufficient.

@miina do you want me to mock this up?

**Comment by miina** [comment]
@jwold Thanks for asking, probably no need for mock at this moment -- the controls would probably have the default Gutenberg design, perhaps we could just verify with you after development if everything looks OK? Thoughts?

**Comment by jwold** [comment]
That's perfect. Thanks @miina!

**Comment by alcurrie** [comment]
@miina  we are working on the updates to the overall interface, so just confirming that you do not need a mock for either of the AC items for this i.e. , A_C-1 - ability to customize font faces_, and _AC-2 change default for images to have “caption” blocks turned off by default and add an interface to turn them back on_  and that you're unblocked on this issue.

**Comment by miina** [comment]
@alcurrie That's correct, the ticket is unblocked, thanks for checking.

**Comment by alcurrie** [comment]
@miina  -  moved AC1: to a separate issue #1643

**Comment by jacobschweitzer** [comment]
This is ready for a code review when you've got a chance @miina : 
https://github.com/Automattic/amp-wp/pull/1663
Thanks for your help getting me started on this!

**Comment by alcurrie** [comment]
Images are now confirmed to not show/display with captions by default. However, now there is no way to add a caption to an image.  (NB: ideally the caption should be a block option. However, it could also, just display the 'image caption'. Neither works currently.  https://cl.ly/bd57277d4b06'
Also, the image update interface does not seem to be clearing/closing once the images are updated.

**Comment by jacobschweitzer** [comment]
Hi @alcurrie the option is a toggle under AMP Story settings section labeled "Show or hide the caption" . I could see it in the video so I know it's not missing 😄 Maybe we could re-word it or put it in another place to make it easier to find? What do you think?

**Comment by alcurrie** [comment]
Hi @jacobschweitzer  - thanks for the quick response.  I'll take another look!

**Comment by alcurrie** [comment]
@jacobschweitzer  - I just took another look. I'm not seeing where the 'AMP Story settings section labeled "Show or hide the caption" is. Any chance you could post a screen shot for me, or describe steps to find it.  https://cl.ly/83d4598b808e

**Comment by miina** [comment]
@alcurrie Note that the Fill Image can never have a caption -- that's because the Fill Image takes the whole room of the background so there is no room for the caption anymore. Other image blocks should have a caption option, could you try with any other Image block?

Do you think it would make sense to have the caption for the Background Fill Image as well?

**Comment by jacobschweitzer** [comment]
Thanks @miina I forgot about that detail related to the Fill Image, it won't show up there right now. Here is where it is located @alcurrie 
![Image caption toggle](https://cl.ly/00b3f48696d1/%255Ba67c1eec05c57fdff88505d25c3e4957%255D_Image%2525202018-12-03%252520at%25252010.13.20%252520AM.png)

**Comment by alcurrie** [comment]
Confirmed that the image caption is turned off by default but works as expected if 'turned on' in AMP stories interface.  https://cl.ly/fdae55a78e9c
Also, shows correctly on Preview https://cl.ly/c016bbfff13d and when published https://cl.ly/af89ad38c519
